components/treelist/toolbar.md

```diff
@@ -158,15 +158,24 @@ You can add your own HTML and components to create a more complex layout in the
 <TelerikTreeList Data="@Data" Pageable="true" ItemsField="@(nameof(Employee.DirectReports))" Width="850px">
 
     <TreeListToolBar>
-        <div style="background:yellow">
-            <TreeListCommandButton Command="Add" Icon="add">Add Employee</TreeListCommandButton>
-        </div>
-        <div style="background: green;">
-            <TelerikDropDownList Data="@( new List<string>() { "first", "second", "third" } )" TValue="string" TItem="string" ValueChanged="@( (string itm) => result = itm )"></TelerikDropDownList>
-        </div>
-        <div style="border: 1px solid red;">
-            <div style="float:right;"><button @onclick="@( () => result = $"Custom button click on {DateTime.Now}"  )">Click me</button></div>
-            <div style="clear:both;"></div>
+        <div style="display: block; flex-grow: 1;">
+            @* the first level children in the toolbar get display: inline-flex and flex-shrink: 0 inherited from the grid,
+                we change it here to show we can, or you can work with the layout the grid defines if it suits your needs *@
+
+            <div style="background:yellow">
+                <TreeListCommandButton Command="Add" Icon="add">Add Employee</TreeListCommandButton>
+            </div>
+            <div style="background: green;">
+                <TelerikDropDownList Data="@( new List<string>() { "first", "second", "third" } )" TValue="string" TItem="string" ValueChanged="@( (string itm) => result = itm )"></TelerikDropDownList>
+            </div>
+
+            @* one example of aligning content to the right with flex *@
+            <div style="border: 1px solid red; justify-content: end;">
```

Comment:
One minor suggestion: `justify-content: end;` is not needed/working here as the parent of this element is displayed as `block`

Reply:
Ah, yes, great catch, that's a copy-paste error  - a remnant of an earlier test I made. Fixed https://github.com/telerik/blazor-docs/pull/222/commits/78cc2963733580709157ab9d1aa45444df7fb5d8